### ISSUE TITLE
Renaming model group causes crash

### ISSUE THREAD

**Issue by rudybrian** [smeighan/xLights#337]
This was observed with xLights 4.2.1 on OSX Yosemite (10.10.5).

Procedure:
* Existing model group with models renamed from "Floods" to "Outside Floods" in Layout->Select Model Groups->Edit Groups
* Rename then select Update Group
* The new name appears in the Model Group list.
* Switch to The Sequencer tab and select Render All
* An error is displayed that Floods does not exists
* App crashes

I can provide the debug report via Dropbox if it is helpful.

**Comment by stevegiron** [comment]
Prevent the crash by:
* Copy all effects on Floods in the sequencer
* Make NEW model group with new name
* Add new group to sequence
* Paste effects to new group
* Remove old group from sequence
* Render, save and enjoy.

**Comment by rudybrian** [comment]
Thanks Steve. Yep, that would work around the renaming issue.

I also noticed that when I re-launched xLights and re-loaded my sequence it complained about the missing "Floods" group, but allowed me to select the new "Outside Floods" group and I was off and running again. 

Another odd thing is that it doesn't seem to have actually saved some of the last set of changes I made to the sequence and I know I saved multiple times during those last edits. Either I am getting forgetful or that could be another bug.
